Tasks/Gradle/gradletask.ts

```diff
@@ -80,8 +106,15 @@ if (javaHomeSelection == 'JDKVersion') {
         var envName = "JAVA_HOME_" + jdkVersion.slice(2) + "_" + jdkArchitecture.toUpperCase();
         specifiedJavaHome = tl.getVariable(envName);
         if (!specifiedJavaHome) {
-            tl.error('Failed to find specified JDK version. Please make sure environment variable ' + envName + ' exists and is set to the location of a corresponding JDK.');
-            tl.exit(1);
+            if (isWindows) {
+                // attempt to discover java home property from registry on Windows
+                specifiedJavaHome = readJavaHomeFromRegistry(jdkVersion, jdkArchitecture);
+            }
+
+            if (!specifiedJavaHome) { 
+                tl.error('Failed to find specified JDK version. Please make sure environment variable ' + envName + ' exists and is set to the location of a corresponding JDK.');
+                tl.exit(1);
```

Comment:
Will make this change in master.  This is a hotfix I don't want to touch too much code to increase testing surface.